Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -9403,6 +9404,13 @@ def generic_http_request(method,
                         timeout=timeout
                         )
 
+    if files:
+        """
+        Need to load the files to memory by read the files (f.read())
+            to be able to send large files without getting a timeout error.
```

Comment:
```suggestion
        To prevent timeout errors when sending large files, it is necessary to load the files into memory by reading them using the command f.read().
```